src/Day02.kt

```diff
@@ -35,13 +21,7 @@ fun main() {
         return totalPaper
     }
 
-    fun getSmallestPerimeter(l: Int, w: Int, h: Int): Int {
-        val measurementSet        = intArrayOf(l, w, h)
-        val indexOfMax            = measurementSet.indexOf(maxOf(l, w, h))
-        val measurementPair       = remove(measurementSet, indexOfMax)
-
-        return measurementPair[0] + measurementPair[0] + measurementPair[1] + measurementPair[1]
-    }
+    fun getSmallestPerimeter(l: Int, w: Int, h: Int): Int = 2 * minOf(l + w, w + h, l + h)
 
     fun getVolumeCubed(l: Int, w: Int, h: Int): Int { return l*w*h }
```

Comment:
You have more code that I didn't modify - but you could consider taking my approach here and applying it to your other solution to make sure that you understand it.